metals/src/main/scala/scala/meta/internal/metals/JarTopLevels.scala

```diff
@@ -230,7 +230,7 @@ final class JarTopLevels(conn: () => Connection) {
     val statement1 = conn().prepareStatement("truncate table toplevel_symbol")
     statement1.execute()
     val statement2 =
-      conn().prepareStatement("truncate table type_hierarchy_jar")
+      conn().prepareStatement("truncate table type_hierarchy")
```

Comment:
Ideally, we would have those table names described in one place, but that can be a follow up